apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -20,6 +20,7 @@ export const nextAuthOptions: NextAuthOptions = {
   providers: [
     EmailProvider({
       server: {
+        // @ts-ignore
```

Comment:
TS started complaining about this and I couldn't figure out why. Should probably just add the SMTP host/port to the ENV and forget this shortcut